test/unit/os/shell_sxe_sxq_spec.lua

```diff
@@ -0,0 +1,43 @@
+-- not all operating systems support the shell_build_argv tests, as of yet.
```

Comment:
Is there any OS where this test doesn't/shouldn't work (even if Windows is not working ATM)?
If not this check could be removed, and the test merged into shell_spec.lua.